src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -313,6 +313,17 @@ export const NavBar: React.FC<NavBarPropsType> = ({
                 <div>{navbarTitle}</div>
               )}
 
+              <div
+                className="nav-schedule nav-photobooth"
```

Comment:
Easy win, some times no need of styles, but here really easy to update the SCSS also

```suggestion
                className="NavBar__photoboot-button nav-schedule nav-photobooth"
```

Reply:
photoboot-button = photoboot**h**-button ;-)

Reply:
fixed in cab515897d168494475f82a5a0970d9270d88094